extensions/localPublish/src/index.ts

```diff
@@ -56,7 +54,7 @@ class LocalPublisher implements PublishPlugin<PublishConfig> {
   public description = 'Publish bot to local runtime';
   static runningBots: { [key: string]: RunningBot } = {};
   private composer: IExtensionRegistration;
-
+  private lastUsedPort = 3979;
```

Comment:
Basically we are using an atomic variable to sync all the threads there, seems like a less breaking change, than making changes in how those tasks are scheduled. 

Looks good to me, what do you think @srinaath